spring-native-image/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think we should better use the short version of license header. According to @spavlusieva:

```
Copyright © [year,] year, Oracle and/or its affiliates.
Released under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl/ or Apache License 2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose either license.
```

Reply:
Hi @olyagpl, I updated the copyright notice to the new concise UPL license copyright notice recommended by @spavlusieva.

Reply:
And we don't change the copyright notices on thirty party files (Maven Wrapper files).